src/javascript/app_2/Stores/common-store.js

```diff
@@ -26,6 +26,19 @@ export default class CommonStore extends BaseStore {
         this.is_socket_opened = is_socket_opened;
     }
 
+    @action.bound
+    setNetworkStatus(status, is_online) {
+        if (this.network_status.class) {
+            this.network_status = Object.assign(
```

Comment:
Maybe for consistency we can use the object spread instead of assign here since we are mostly using that in other places?

Reply:
@OskarAhl They don't do the same thing though. With `Object.assign` we can keep the reference and only change the value, but if we use spread syntax, it will change the reference.

Reply:
Ah I see now, so the purpose is to only update the class and tooltip value - then why not update it this way?
```suggestion
            this.network_status.class = status.class;
            this.network_status.tooltip = status.tooltip;
```

Reply:
@OskarAhl Yeah, for clarity we can do that. Thanks.